packages/react-components/src/AccessibilityScenarios/Menu2.stories.tsx

```diff
@@ -0,0 +1,53 @@
+import * as React from 'react';
+
+// https://github.com/microsoft/fluentui/pull/18695#issuecomment-868432982
+// eslint-disable-next-line @typescript-eslint/ban-ts-comment
+// @ts-ignore
+import { MenuButton } from '@fluentui/react-button';
+
+// https://github.com/microsoft/fluentui/pull/18695#issuecomment-868432982
+// eslint-disable-next-line @typescript-eslint/ban-ts-comment
+// @ts-ignore
+import { Menu, MenuTrigger, MenuList, MenuPopover, MenuItem, MenuSplitGroup } from '@fluentui/react-menu';
+
+import { Scenario } from './utils';
+
+export const MenuWithSplitItemAccessibilityScenario: React.FunctionComponent = () => {
+  return (
+    <Scenario pageTitle="Menu with split item">
+      <Menu>
+        <MenuTrigger>
+          <MenuButton>More actions</MenuButton>
+        </MenuTrigger>
+
+        <MenuPopover>
+          <MenuList>
+            <MenuItem>New Window</MenuItem>
+            <MenuItem>Create Folder</MenuItem>
+            <Menu>
+              <MenuSplitGroup>
+                <MenuItem>Open</MenuItem>
+                <MenuTrigger>
+                  <MenuItem id="nestedTrigger">More open options</MenuItem>
```

Comment:
please move the content ("More open options") to aria-label of the MenuItem

Reply:
Done.